src/main/resources/citizen/templates/document/claimIssueReceipt.html

```diff
@@ -513,52 +514,56 @@
     <table>
       <tr>
         <td></td>
-        {% if claim.statementOfTruth is not defined or claim.statementOfTruth is empty %}
         <td>
           I believe that the facts stated in this claim form are true.
         </td>
-        {% else %}
-        <td>
-          The claimant believes that the facts stated in this claim are true.
-        </td>
-        {% endif %}
       </tr>
       {% if claim.statementOfTruth is not defined or claim.statementOfTruth is empty %}
-      <tr>
+      <tr class="no-border">
         <td></td>
         <td>
           {{ claimant.fullName }}
         </td>
       </tr>
-      <tr>
+      <tr class="no-border">
         <td></td>
         <td>
           {{ claim.issuedOn }}
         </td>
       </tr>
       {% endif %}
       {% if claim.statementOfTruth is defined and claim.statementOfTruth is not empty %}
-      <tr>
+      <tr class="no-border">
+        <td></td>
+        <td>
+          <strong>Company:</strong>
```

Comment:
should this not be in an If statement? From my reading of this company will always be present even if the person signing it is an individual?